packages/@aws-cdk/aws-lambda-python/lib/bundling.ts

```diff
@@ -0,0 +1,78 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as lambda from '@aws-cdk/aws-lambda';
+import * as cdk from '@aws-cdk/core';
+import { findUp } from './util';
+
+/**
+ * Options for bundling
+ */
+export interface BundlingOptions {
+  /**
+   * The root of the project. This will be used as the source for the volume
+   * mounted in the Docker container. If you specify this prop, ensure that
+   * this path includes `entry` and any module/dependencies used by your
+   * function otherwise bundling will not be possible.
+   *
+   * @default - the closest path containing a .git folder
+   */
+  readonly projectRoot?: string;
```

Comment:
Why do you need to mount the whole CDK project here? Isn't the assumption here that the Lambda is self contained in a directory with the handler and its requirements?